src/css/styles/_c-lang.scss

```diff
@@ -0,0 +1,63 @@
+.c-lang {
+  position: absolute;
+  top: 1rem;
+  right: 0;
+
+  &__toggle {
+    cursor: pointer;
+  }
+
+  &__toggle-text {
+    padding: 1rem 0.5rem 0.5rem !important;
+  }
+
+  &__icon,
+  &__icon-hide {
+    max-width: 12px;
+  }
+
+  &__dd {
+    position: absolute;
+    top: 1.7rem;
+    padding: 0;
+    list-style: none;
+    background: transparent;
+  }
+
+  &-dd__option {
```

Comment:
`.c-lang-dd__option` It should not be in "scope" of `.c-lang`

- [ ] move `c-lang-dd__option` out of `c-lang`

Reply:
General - please do not use `&-name` (files: `_c-lang.scss`, `_c-list.scss`, `_c-num-list.scss`)

- [ ] remove `&-name`

Reply:
- [ ] Please do not use `block--modifer__element`

Examples:
* `c-nav--mobile__menu`
* `c-menu--mobile__list-item`
* `c-menu--mobile__link`
* `c-lang--movile__item`

Reply:
Do we use `c-contact__name-txt`?

remove `c-contact__name-txt`